shell/lib/buildx.sh

```diff
@@ -5,6 +5,7 @@
 # BUILDX_VERSION is the version of 'docker buildx' we should use if not
 # already installed on the host system.
 BUILDX_VERSION="v0.11.2"
+APPNAME="$1"
```

Comment:
If we do keep this, please document it using the format above, example:

```
# APPNAME is xyz and is used for the buildx builder's name.
```

This helps with development because there are things like [bash lsp](https://github.com/bash-lsp/bash-language-server) that can read this and make it available on hover, for example.

Reply:
Simplified, no need for comment anymore